Client/Frontend/Menu/BookmarksViewController.swift

```diff
@@ -180,6 +180,12 @@ class BookmarksViewController: SiteTableViewController {
       print(error.description)
     }
     
+    if tableView.isEditing == false && bookmarksFRC?.fetchedObjects?.count == 0 {
+        editBookmarksButton?.isEnabled = false
```

Comment:
Since this is just using a boolean conditional for a value setter

e.g.
```
if foo
    bar = foo
else 
    bar = foo
```

We can just always perform `bar = foo`:

```
editBookmarksButton.isEnabled = bookmarksFRC?.fetchedObjects?.count != 0